test-client/Client/ClientConfigurationTest.cs

```diff
@@ -0,0 +1,22 @@
+using NUnit.Framework;
+using Moq;
+using Statful.Core.Client.Buffer;
+using Statful.Core.Client.Client;
+using Statful.Core.Client.Configuration;
+using Statful.Core.Client;
+using System;
+using System.IO;
+
+namespace test_client.Client
+{
+    public class ClientConfigurationTest
+    {
+        [Test]
+
+        public void Test_client_configuration_from_file(){
+            IStatfulClient client = StatfulClientFactory.CreateStatfulClient("/home/francisco/IdeaProjects/statful-dotnet-core-client/test-client", "appsettings.json");
```

Comment:
can we make it work with relative path? otherwise this test will only work on your machine and everyone that needs to run this test will have to edit.

Reply:
My mistake, did not intend to push this as it was